index.js

```diff
@@ -4,34 +4,176 @@ const closebtn = document.querySelector('.close-btn');
 const portfoliolink = document.querySelector('.portfolio-link');
 const aboutlink = document.querySelector('.about-link');
 const contactlink = document.querySelector('.contact-link');
-let menuOpen = false;
+const arrayProjectBtn = document.querySelectorAll('.project-btn');
 
 menubtn.addEventListener('click', () => {
-  !menuOpen && menubtn.classList.add('hidden');
+  menubtn.classList.add('hidden');
   headertoolbar.classList.remove('hidden');
-  (menuOpen = true);
 });
 
 closebtn.addEventListener('click', () => {
-  menuOpen && menubtn.classList.remove('hidden');
+  menubtn.classList.remove('hidden');
   headertoolbar.classList.add('hidden');
-  (menuOpen = false);
 });
 
 portfoliolink.addEventListener('click', () => {
   headertoolbar.classList.add('hidden');
   menubtn.classList.remove('hidden');
-  console.log('click link'); /* eslint-disable-next-line no-console */
 });
 
 aboutlink.addEventListener('click', () => {
   headertoolbar.classList.add('hidden');
   menubtn.classList.remove('hidden');
-  console.log('click link'); /* eslint-disable-next-line no-console */
 });
 
 contactlink.addEventListener('click', () => {
   headertoolbar.classList.add('hidden');
   menubtn.classList.remove('hidden');
-  console.log('click link'); /* eslint-disable-next-line no-console */
-});
\ No newline at end of file
+});
+
+const dataCards = [
+  {
+    id: 0,
+    url: './img/newnew.svg',
+    title: 'Keeping track of hundreds of components',
+    description: `Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the
+  industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type
+  and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and
+  typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,
+  when an unknown printer took a galley of type and scrambled it 1960s with the relea`,
+    tecnologies: {
+      tech1: 'Rubi on rails',
+      tech2: 'css',
+      tech3: 'JavaScript',
+    },
+    linkVersion: 'https://www.google.com',
+    linkSource: 'https://adarachel.github.io/personal-portfolio/',
+  },
+  {
+    id: 1,
```

Comment:
Sorry reviewer, but i do not understand the request/what you mean... please explain better. thank you

Reply:
Hi @adarachel, when running it on my machine the projects which are displayed are the projects you hard coded in your html and not the one from your javascript

Reply:
![image](https://user-images.githubusercontent.com/108772486/195429312-b2ff9864-9cc6-443a-a2dd-30f04709c635.png)

You must be running it the wrong way because as you can see in the screenshot below, i am having no issues viewing the projects coded in my JS file. You are supposed to click on the "see project" button. Please get back to me, thanks.